### PR TITLE
Create issue templates.

### PR DESCRIPTION
**Pull request by jheer**

### CODE REVIEW

**Comment by nyurik** [comment]
thanks for linking to the gist, I will keep it up to date (e.g. will improve instructions as needed)

**Comment by domoritz** [comment]
I think the question template can be a bit more explicit that we reserve GitHub issues for bugs and feature requests.

File: .github/ISSUE_TEMPLATE/questions-or-help.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 ---
 name: Questions or Help
-about: For general questions, please use our Slack channel or mailing list.
+about: Do NOT post usage questions here. Please use Slack or our mailing list instead.
```
Comment (nyurik) [inline .github/ISSUE_TEMPLATE/questions-or-help.md]: Should there be a link to Slack invitation, and possibly mention stackoverflow?  Mailing list is great, but it is not very well searched by google, and tend not to show up in query results, making people ask the same questions.

Comment (nyurik) [reply .github/ISSUE_TEMPLATE/questions-or-help.md]: P.S. Assuming links will work in the `about` section.  If not, the links below are good too.

**Pull request merged**